typhos/tests/test_console.py

```diff
@@ -17,59 +17,59 @@ def test_base_console(qtbot):
     tc.shutdown()
 
 
-def test_add_happi_device(qapp, qtbot, happi_cfg, client):
-    device = client['Syn:Motor'].get()
+# def test_add_happi_device(qapp, qtbot, happi_cfg, client):
```

Comment:
Maybe
```
# top-level module:
import pytest

pytest.skip()
```

Reply:
I was hoping that I might find the solution to these test suite problems, but if I give up I'll `pytest.skip()`